.npmignore

```diff
@@ -0,0 +1,11 @@
+.eslintcache
```

Comment:
This file is not actually needed. Since `package.json` explicitly says to pack only the `bin` and `src` folders, all other files will be ignored. Please remove.